plugins/jenkins/app/models/samson/jenkins.rb

```diff
@@ -6,9 +6,25 @@ class Jenkins
     URL = ENV['JENKINS_URL']
     USERNAME = ENV['JENKINS_USERNAME']
     API_KEY = ENV['JENKINS_API_KEY']
+    ERROR_COLUMN_LIMIT = 255
 
     attr_reader :job_name, :deploy
 
+    def self.deployed!(deploy)
+      return unless deploy.succeeded?
+      deploy.stage.jenkins_job_names.to_s.split(/, ?/).map do |job_name|
```

Comment:
the old code made sure `jenkins_job_names` was present, should that still happen?